src/flatc.cpp

```diff
@@ -342,6 +357,17 @@ int main(int argc, const char *argv[]) {
                     " for " +
                     filebase);
             }
+            if(grpc_enabled) {
+              if(generators[i].generate_grpc == nullptr) {
+                Error(std::string("Unable to generate GRPC interface") +
```

Comment:
instead give error here: "GRPC interface generation not implemented for this language"